msg/Person.msg

```diff
@@ -1,21 +1,20 @@
-Header header                   # standard header
-
-uint32 id						# unique identifier to a person
-	
-geometry_msgs/PoseStamped pose	# real pose of a detected person
-
-geometry_msgs/PoseStamped safe_pose		# safety pose for a detected person (to approach by base)
-
-sensor_msgs/Image rgb_image                     # image cropped to the bounding box of the detected person
-
-float32 height					# measured height of the person
-
-float32 width					# measured width of the person 
-
-float32 depth					# measured depth of the person
-
-bool is_tracked					# is the user tracked
-
-bool is_occluded				# is the user occluded 
-
-float32 probability				# probability of beeing a person at this pose
+string identity
```

Comment:
what are you considering as identity? could this be different than name?

Reply:
I thought the names may not always be available, e.g. crowd of unknown people or operator. It can also be that two people have the same name. Identity is maybe the more general word for identifying people.

Reply:
I see. It's still good to add a name (if we have one), the identity can be a unique identifier (help differentiate between two people with the same name).

Reply:
The idea was to use names as identity if that make sense, otherwise it would be some unique identifier. Other knowledge about the person, (e.g. given name) may be drawn from the knowledge base. I think the `name` field would be redundant with `identity` and doesn't add much in term of functionality.

Reply:
I disagree, the name is not unique but the identifier may be. It does add an additional field to the message, but given the way ROS msgs work anyways, they are optional. In cases where we have unique names, they could be indeed redundant, but I would keep these two as separate things.

Reply:
OK sure I can add it. May also be useful for cases when the robot asks people for their names.